docs/pages/quick-start.mdx

```diff
@@ -127,7 +127,6 @@ The client package will vary depending on which template used (vanilla, react, p
 
 - You can adjust `createClientComponents.ts` to either override contract components or add client-only components.
 - If you are using chains other than foundry/anvil and lattice testnet, you can add them in `getNetworkConfig.ts`
-- `createSystemCalls` represents how the client talks to the system contracts via our `worldSend` helper
```

Comment:
fwiw we still have and use `createSystemCalls`, we just go through `worldContract.write` instead